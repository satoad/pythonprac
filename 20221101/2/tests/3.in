a = Triangle((0, 0), (3, 0), (0, 3))
b = Triangle((0.01, 0.01), (0.2, 0.1), (0.1, 0.2))
print(*(f"{n}({bool(x)}):{round(abs(x), 3)}" for n, x in zip("ab", (a, b))))
print(b & a)
print(b in a)
print(b < a)