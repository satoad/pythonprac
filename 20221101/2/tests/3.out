a(True):4.5 b(True):0.014
True
True
True
